stubs/zstd/zstd.pyi

```diff
@@ -0,0 +1,15 @@
+from _typeshed import ReadableBuffer
+
+class Error(Exception): ...
+
+def ZSTD_compress(data: ReadableBuffer, level: int | None, threads: int | None) -> bytes: ...
```

Comment:
```suggestion
def ZSTD_compress(__data: ReadableBuffer, __level: int = ..., __threads: int = ...) -> bytes: ...
```
All arguments are positional-only since the code uses PyArg_ParseTuple, which works only with positional args. (This also applies to the other functions.)

Also, the last two arguments are optional but do not allow None.